### PR TITLE
Upstream sync 31

### PR DESCRIPTION
**Pull request by bamachrn**
**What this PR does / why we need it**:
Code sync up with upstream release-3.1 branch

**Special notes for your reviewer**:
We have kept the changes for internal OWNERS and Makefile

### CODE REVIEW

File: internal/tlsutil/tlsutil_test.go [hunk 1]
```diff
@@ -54,32 +53,63 @@ func TestClientConfig(t *testing.T) {
...
+	if cfg.InsecureSkipVerify {
+		t.Fatalf("insecure skip verify mistmatch, expecting false")
+	}
+	if cfg.RootCAs != nil {
+		t.Fatalf("mismatch tls RootCAs, expecting nil")
 	}
-	return path
 }
 
 func TestNewClientTLS(t *testing.T) {
```
Comment (pmacik) [inline internal/tlsutil/tlsutil_test.go]: @bamachrn This `func TestNewClientTLS` declaration is duplicated here.

Comment (bamachrn) [reply internal/tlsutil/tlsutil_test.go]: thanks @pmacik seems like I kept both of them while solving the merge conflict. solved now

**Pull request closed without merging**